integration-tests/zendeskRequestAPI.spec.ts

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios'
+
+describe('Zendesk request API', () => {
+  it('returns a Invalid request source on invalid Zendesk Webhook Signature', async () => {
+    const awsBaseUrl = process.env.AWS_BASE_URL as string
+    const invalidSignature = 'cCxJHacr678ZZigFZZlYq4qz2XLWPEOeS+PPDuTivwQ='
+
+    const data = {
+      zendeskId: '408',
+      resultsEmail: 'user@test.gov.uk',
+      resultsName: 'test user',
+      dateFrom: '2022-09-06',
+      dateTo: '2022-09-06',
+      identifierType: 'journey_id',
+      sessionIds: '',
+      journeyIds: '098 565 2213',
+      userIds: '',
+      eventIds: '',
+      piiTypes: 'name',
+      dataPaths: ''
+    }
+
+    const headers = {
+      Host: awsBaseUrl,
+      'X-Zendesk-Webhook-Signature': invalidSignature
+    }
+
+    const invalidRequestError = async () => {
+      return axios({
+        url: `https://${awsBaseUrl}/Stage/zendesk-webhook`,
+        method: 'POST',
+        headers,
+        data
+      })
+        .then(() => {
+          throw 'Axios did not recieve an error'
+        })
+        .catch((error) => {
+          if (error.response) {
+            return error.response
+          }
+          throw 'Axios did not recieve a response error'
+        })
+    }
+
+    const errorResponse = await invalidRequestError()
+    expect(errorResponse.status).toBe(400)
```

Comment:
The acceptance criteria states unauthorised, suggesting a 401. Do we need to update the a/c? Also, there's a final a/c about the attempt being logged, not sure if that should be in Cloudwatch?

Reply:
We could - a 401 suggests you should re-authenticate which isn't how this works though.  I think a 400 is probably right for whats actually happening in this case.